src/applications/gi/components/profile/AdditionalInformation.jsx

```diff
@@ -111,6 +111,19 @@ export class AdditionalInformation extends React.Component {
           &nbsp;
           {it.independentStudy ? 'Yes' : 'No'}
         </div>
+        <div>
```

Comment:
does this need a production flag?

Reply:
Hey you are correct for both, adding the flags now